### ISSUE TITLE
[Question] Is it possible to use hooks from react-router package?

### ISSUE THREAD

**Issue by siarheipashkevich** [salvoravida/redux-first-history#52]
Hi @salvoravida 

Thanks for your great wrapper and quick support of the rr6.

1) In rr we have some hooks for using, can I use these hooks in my project (redux) or I should create my own hook where I should call `push` method from you wrapper?
2) If I wanna to do anything I should touch your api methods first for the correct work of the rr and wrapper?

Thanks

**Comment by salvoravida** [comment]
Sure you can use rr official hooks! ( when at the end they call history.push, redux first history tunnel translate it into dispatch push)

**Comment by siarheipashkevich** [comment]
But in the documentation we have one point:
```
use push action creator from redux-first-history if you need to dispatch location from saga or connected components.
```

But if I will call hooks from rr directly it's will be handled correctly for redux state?

Please ignore 2 question it's the same with first one, sorry )

**Comment by salvoravida** [comment]
Yes, but that case is when you need push from redux-saga for example or the old react-redux "connect" hoc.

Yes, now you can use useHistory from react-rotuer directly, it will works with redux state!

**Comment by siarheipashkevich** [comment]
Thanks a lot.
